src/index.js

```diff
@@ -59,26 +62,40 @@ const stepTrends = $('#step-trends');
 const stepGoalChart = $('#step-goal-chart');
 const friendList = $('#friend-list');
 
-const randomId = Math.floor(Math.random() * (50 - 1) + 1);
-const userRepository = new UserRepository(userData, randomId);
-const hydrationRepository = new HydrationRepository(hydrationData, randomId);
-const sleepRepository = new SleepRepository(sleepData, randomId);
-const activityRepository = new ActivityRepository(randomId, activityData);
-const user = new User(userRepository.getUserData());
-
-updateUserDataDOM(userRepository.getUserData());
-compareStepGoal(userRepository.getUserData());
-displayDailyOz();
-displayWeeklyOz();
-displayBestSleepers();
-displayCurrentDate(getCurrentDate());
-displaySleep();
-displayActivity();
-displayAverageWeeklyActivity();
-displayWeeklyActivity();
-friendActivityData(getCurrentDate());
-displayTrends();
-displaySleepChart()
+var randomId;
+var userRepository;
+var hydrationRepository;
+var sleepRepository;
+var activityRepository;
+var user;
+
+Promise.all([userData]).then(element => {
+  userData = element[0]
+  randomId = Math.floor(Math.random() * (50 - 1) + 1);
+  userRepository = new UserRepository(userData, randomId);
+  hydrationRepository = new HydrationRepository(hydrationData, randomId);
+  sleepRepository = new SleepRepository(sleepData, randomId);
+  activityRepository = new ActivityRepository(randomId, activityData);
+  user = new User(userRepository.getUserData());
+}).then(() => {
+  displayAllData()
+});
```

Comment:
@VeeAndrade @sertmer @benjamin-firth Don't forget, you need to have error handling for your fetch requests. Look more into Promise.all and think about where makes the most sense to put a `.catch()`